submissions/Vlada-khom/html-css-popup/styles.css

```diff
@@ -0,0 +1,185 @@
+body {
+  background-color: rgba(0, 255, 255, 0.2);
+}
+
+header nav {
+  text-align: right;
+  font-family: "Roboto";
+}
+header nav .navLink {
+  vertical-align: middle;
+  display: inline-block;
+  padding: 10px;
+  text-decoration: none;
+  color: black;
+}
+header nav .navLink .imgLink {
+  width: 30px;
+  height: 30px;
+}
+header nav .navLink .avatar {
+  border-radius: 50%;
+}
+header nav .burger {
+  position: relative;
+}
+header nav .burger .input {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  clip: rect(0, 0, 0, 0);
```

Comment:
For hiding checkboxes use full version of visually-hidden:
```CSS
.visually__hidden {
    position: absolute;
    overflow: hidden;
    clip: rect(0 0 0 0);
    height: 1px;
    width: 1px;
    margin: -1px;
    padding: 0;
    border: 0;
}
```
Or watch [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) find other solutions.